capstone/src/main/java/com/google/univiz/api/MapData.java

```diff
@@ -1 +1,13 @@
 package com.google.univiz.api;
+
+/*
+ * The MapData interface is responsible for all data management/processing specific for the maps
+ * feature of the UniViz webapp.
+ */
+public interface MapData {
+  /*
+   * Takes a list of college names and returns a CollegeMapsData object to the caller. This object
```

Comment:
Should we use a more stable identifier for colleges? Something like a unique id (could be scorecard uid) instead of name.

Reply:
hmm ok-so the search servlet would probably be able to provide that-a name + a uid?

Reply:
done!

Reply:
Can you update the documentation to match the signature of the method?